python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+    root1.geometry("7000x6000")
+    root1.minsize(500,600)
+    canvas=Canvas(root1,width=2000,height=2200,bg="sky blue")
+    canvas.place(x=0,y=0)
+    M=StringVar()
+    x_1,y_1,x_2,y_2=550,200,650,300  #coordinates for the big circle
+    a_1,b_1,a_2,b_2=730,220,790,280  #coordinates for the small circle
+    global countsmallclick,countbigclick
+    #global bigclick,smallclick
+    bigclick=0
+    smallclick=0
+    if charge1val>charge2val:      
+        oval1=canvas.create_oval(x_1,y_1,x_2,y_2,fill="orange")        
+        oval2=canvas.create_oval(a_1,b_1,a_2,b_2,fill="grey")
+        text1=canvas.create_text(600,250,text=f"{charge1val}",fill="white")
+        text2=canvas.create_text(760,250,text=f"{charge2val}",fill="white")
+        line1=canvas.create_line(650,250,730,250,fill="black",width="3")
+        f,g=compute.mp(650,250,730,250)
+        dist=int((compute.liner(650,250,730,250)//80))
+        distance=canvas.create_text(f,g,text=f"{dist}")
+        flag=1
+        countbigclick=0
+        countsmallclick=0     
+    elif charge2val>charge1val:
+        charge1val,charge2val=charge2val,charge1val
+        oval1=canvas.create_oval(x_1,y_1,x_2,y_2,fill="orange")        
+        oval2=canvas.create_oval(a_1,b_1,a_2,b_2,fill="grey")
+        text1=canvas.create_text(600,250,text=f"{charge1val}",fill="white")
+        text2=canvas.create_text(760,250,text=f"{charge2val}",fill="white")
+        line1=canvas.create_line(650,250,730,250,fill="black",width="3")
+        countbigclick=0
+        countsmallclick=0
+        f,g=compute.mp(650,250,730,250)
+        dist=int((compute.liner(650,250,730,250)//80))
+        distance=canvas.create_text(f,g,text=f"{dist}") 
+        flag=1        
+    else:
+        oval1=canvas.create_oval(610,220,670,280,fill="grey")
+        text1=canvas.create_text(640,250,text=f"{charge1val}",fill="white")        
+        oval2=canvas.create_oval(730,220,790,280,fill="grey")
+        text2=canvas.create_text(760,250,text=f"{charge2val}",fill="white")
+        line2=canvas.create_line(670,250,730,250,fill="black",width="3")
+        countbigclick=0
+        countsmallclick=0
+        f,g=compute.mp(670,250,730,250)
+        dist=int((compute.liner(670,250,730,250))//60) 
+        distance=canvas.create_text(f,g,text=f"{dist}")
+        flag=2      
+    bumbam=i.get()
+    bambum=mango.get()
+    q=compute.getter(bambum)
+    force=compute.forcecalc(charge1val,charge2val,q,dist,bumbam)
+    M.set(str(force))
+    lavel=Label(root1,textvariable=M,font="TIMESNEWROMAN 12 bold",bd=3,height=1,fg="green") 
+    lavel.place(x=865,y=600)
+    lav=Label(root1,text="The Electrostatic Force Of Attraction is :",font="TIMESNEWROMAN 12 bold",bg="white",fg="red")
+    lav.place(x=550,y=600)
+    def dad(event):#radius of bigcircle=50, radius of smaller circle=30
+        global x11,x_11
+        global y11,y_11
+        global lavel1,heaven
+        global oval3,oval4
+        global text3,text4
```

Comment:
Codacy found an issue: [Global variable 'text3' undefined at the module level](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)